scripts/build-glean-metadata.py

```diff
@@ -443,9 +451,34 @@ def get_app_variant_description(app):
             )
         )
 
+    # write labels (if any)
+    labels = [
+        {"name": k, "description": v}
+        for (k, v) in annotations_index.get(app.app_name, {}).get("labels", {}).items()
+    ]
+    if labels:
+        app_data["labels"] = labels
+        for label in labels:
+            label_metrics = [
+                metric
+                for metric in app_data["metrics"]
+                if label["name"] in metric.get("labels", [])
+            ]
+            label["metric_count"] = len(label_metrics)
+            open(os.path.join(app_labels_dir, f"{label['name']}.json"), "w").write(
+                json.dumps(
+                    dict(
+                        label,
+                        origin=app.app_name,
+                        metrics=label_metrics,
+                    )
+                )
+            )
```

Comment:
Why do we need to write label data to separate json files? It looks like write the data here but never use it, as the label data needed was pulled from the main `index.json` file rather than `[label].json`

Reply:
Yup, another leftover. Will take this out.